2022-11-16-recommenders-2.md

```diff
@@ -0,0 +1,129 @@
+## DIY Recommendation Systems (Part 2)
+
+* **Goal**: After learning about the different types of recommendation system algorithms during week 1, now we compare the two algorithms we explored. Also, we create a more advanced version of a recommendation system using a hybrid approach.
+* **Dates**: from 16th to 23rd November.
+* **Where:** [`#project-of-the-week`](https://app.slack.com/client/T01ATQK62F8/C02BP4FQH36) in DataTalks.Club (get in slack here: [https://datatalks.club/slack.html](https://datatalks.club/slack.html))
+
+For more information about the "Project of the Week" initiative
+at DataTalks.Club, see [README.md](README.md).
+
+If you want to receive reminders about this event, sign up here
+
+<p align="center">
+  <a href="TODO"><img src="https://user-images.githubusercontent.com/875246/185755203-17945fd1-6b64-46f2-8377-1011dcb1a444.png" height="50" /></a>
+</p>
+
+
+## Technologies 
+
+* Scikit-Learn
+* [LightFM](https://github.com/lyst/lightfm)
+* Jupyter notebooks
+
+
+Note: this is a suggested list of technologies, you can chose
+alternatives instead
+
+## Plan
+
+This is a proposed plan only, you don’t have to follow it day-by-day.
+
+
+### Day 0 (16 November, Wednesday)
+
+* Recap of the project you work on during Part 1 of this project-of-the-week.
+* If you couldn’t work on Part 1 of this project-of-the-week, then you can catch up by quickly reviewing some of the suggested materials.
+* Create a GitHub project
+* Share your progress in Slack and on social media
+
+Suggested materials
+
+* 💻 [Sample project](https://github.com/alexeygrigorev/product-recommendation) for Part 1.
+
+Would you like to add your project for Part 1? Create a PR with links!
```

Comment:
Probably it makes sense to give a link to the part one?

Reply:
I mean turn it into a link =)

Reply:
Great point! We definitely need to add a link to part 1.